documentation/guide/OWSGuide.adoc

```diff
@@ -363,84 +646,9 @@ You will rarely need those.
 
 === Remote Debugging
 
-#<documentation added in the near future>#
-// TODO Janak
-// have a look at the tooltips provided with the ows-settings
-
-=== System Configuration
```

Comment:
We have linked to this topic from the forum so we should not remove it

Reply:
restored